pjsip/src/pjsua-lib/pjsua_media.c

```diff
@@ -3003,6 +3003,15 @@ pj_status_t pjsua_media_channel_create_sdp(pjsua_call_id call_id,
             continue;
         }
 
+        /* Check if request supports PJSIP_INV_REQUIRE_SIPREC. If so
+         * Get label attribute in SDP offer and add label attribute to SDP answer
+         */
+        if (call->inv && (call->inv->options & PJSIP_INV_REQUIRE_SIPREC)) {
+            const pj_str_t STR_LABEL_ATTR = {"label", 5};
+            m->attr[m->attr_count++] = pjmedia_sdp_media_find_attr(
```

Comment:
I think you need to duplicate/clone the attribute, instead of just taking it from the offer, since you can't rely on the offer's lifetime.

Reply:
fix it